Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -50,6 +50,22 @@ class SentryFeedback: NSObject, SentrySerializable {
         return dict
     }
     
+    func dataDictionary() -> [String: Any] {
+        var dict: [String: Any] = [
+            "message": message
```

Comment:
`m`: No UI test is failing when I modify this to the following
```suggestion
            "messag": message
```

I don't think we should add a UI test for this, but instead some unit tests to validate the functionality.